frontend/pages/model/[modelId]/release/new.tsx

```diff
@@ -55,15 +61,47 @@ export default function NewRelease() {
       }
 
       const metadata = filesMetadata.find((fileWithMetadata) => fileWithMetadata.fileName === file.name)?.metadata
-      const postFileResponse = await postFile(file, model.id, file.name, file.type, metadata)
 
-      if (!postFileResponse.ok) {
-        setErrorMessage(await getErrorMessage(postFileResponse))
+      const fileResponse = await axios
```

Comment:
Can we pull most of this logic into a new `postFile` function in `actions/release` and get rid of the old one? We'd just need to expose `onUploadProgress` via a callback param, but other than that I would expect it to be handled the same as any other action at the component level.